capten/common-pkg/plugins/argocd/repositories.go

```diff
@@ -31,6 +37,7 @@ func (a *ArgoCDClient) CreateRepository(ctx context.Context, repo *Repository) (
 				Message: repo.ConnectionState.Message,
 			},
 		},
+		Upsert: repoUpsert,
 	})
```

Comment:
is new project adding gives error if upsert is true ?
if not then, no needed check project exist

Reply:
yes, we won't get error. removed the project existing check by proving upsert value as true.